docs/service-configuration.md

```diff
@@ -0,0 +1,204 @@
+## Configuring a Service through Fabrikate configurations
+
+The goal of this guide is to explain how to configure a single Service's deployment within the High Level Definition repository's configurations.
+
+### Requirements
+To follow this guide, there are some setup requirements:
+- The Service's deployment needs to be templitized as a [Helm](https://helm.sh/) Chart.
+- The Service needs to be added to the High Level Definition repository as a [fabrikate](https://github.com/microsoft/fabrikate) component referencing the above Helm Chart.
```

Comment:
nit: Sometimes service is written as "Service" other times as "service", I think if it's lower case throughout the doc, it's fine.